cmd/anubis/main.go

```diff
@@ -177,7 +177,7 @@ func main() {
 
 	var h http.Handler
 	h = s
-	h = internal.DefaultXRealIP(*debugXRealIPDefault, h)
+	h = internal.RemoteXRealIP(useRemoteAddress, h)
```

Comment:
```suggestion
	h = internal.RemoteXRealIP(*useRemoteAddress, h)
```

[flag](https://pkg.go.dev/flag) will make this always be non-nil